src/CommunityToolkit/Xamarin.CommunityToolkit/PlatformConfiguration/AndroidSpecific/NavigationBar.shared.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using Xamarin.Forms;
+using TargetElement = Xamarin.Forms.Application;
+using XFPC = Xamarin.Forms.PlatformConfiguration;
+
+namespace Xamarin.CommunityToolkit.PlatformConfiguration.AndroidSpecific
+{
+	public static partial class NavigationBar
+	{
+		/// <summary>
+		/// Sets the color of application's navigation bar
+		/// </summary>
+		public static readonly BindableProperty ColorProperty = BindableProperty.Create(
+			"Color", typeof(Color), typeof(TargetElement), Color.Default, propertyChanged: (_, _, newValue) => SetColor((Color)newValue));
+
+		/// <summary>
+		/// Sets the style of application's navigation bar
+		/// </summary>
+		public static readonly BindableProperty StyleProperty = BindableProperty.Create(
+			"Style", typeof(NavigationBarStyle), typeof(TargetElement), NavigationBarStyle.Default, propertyChanged: (_, _, newValue) => SetStyle((NavigationBarStyle)newValue));
+
+		/// <summary>
+		/// Needed for BindableProperty to work. Don't call this method.
+		/// </summary>
+		public static Color GetColor(BindableObject element) => throw new NotImplementedException();
```

Comment:
```suggestion
		[EditorBrowsable(EditorBrowsableState.Never)]
        public static Color GetColor(BindableObject element) => throw new NotImplementedException();
```

Reply:
Turns out this breaks IneliSense for `BindableProperty`. (the same as not including this method at all)